### ISSUE TITLE
Temprarily deactivate

### ISSUE THREAD

**Issue by pacomolinero** [mastodon/mastodon#31861]
### Pitch

Currently there is no possibility to temporarily deactivate the Mastodon account, it can be migrated or deleted, but it cannot be left inactive for a while. It would be an option where no messages are received and no messages can be posted. All account activity would be suspended until it is reactivated again.

### Motivation

There are many people who feel overwhelmed at certain times by social media and need a period of rest but don't want to lose their Mastodon account or their relationships, just put them on pause for a while.

**Comment by mochsner** [comment]
I agree this would be useful! Just a basic banner or description on I just did this, and my current workaround was just to
